spec/mongoid/touchable_spec.rb

```diff
@@ -584,5 +584,118 @@
         end
       end
     end
+
+    context "when the touch option is false" do
+
+      shared_examples "does not update the parent" do
+
+        let!(:start_time) { Timecop.freeze(Time.at(Time.now.to_i)) }
+
+        let(:update_time) do
+          Timecop.freeze(Time.at(Time.now.to_i) + 2)
+        end
+
+        after do
+          Timecop.return
+        end
+
+        let(:building) do
+          parent_cls.create!
+        end
+
+        let(:entrance) do
+          building.entrances.create!
+        end
+
+        before do
+          entrance
+          update_time
+          entrance.touch
+        end
+
+        it "updates the child's timestamp" do
+          entrance.updated_at.should == update_time
+          entrance.reload.updated_at.should == update_time
+        end
+
+        it "does not update the parent's timestamp" do
+          building.updated_at.should == start_time
+          building.reload.updated_at.should == start_time
+        end
+      end
+
+      context "when touch is false on belongs_to" do
+        let(:parent_cls) { TouchableSpec::Referenced::Building }
```

Comment:
For these types of feature-related specs I think we should move way from "real world" class names (Building, Hospital, etc.) and have the class names be things like EmbedsOne, EmbeddedIn, etc. which describe their actual relationship.

Reply:
The problem with that is that the naming space gets too cluttered. How many "NoParent" classes are there going to be, you know? It's also not that big of a deal to just look at the class definition.